### DISCUSSION TITLE
User table list is blank

### DISCUSSION THREAD

**Discussion by brandon-9** [statamic/cms#4425] [Troubleshooting] [answered]
I have set up Statamic on an existing project. 
The user table displays blank, however, it is aware of the users as it displays the pagination.

If I click my profile, I view all my blueprint fields, all of which pulled the content from the database correctly.

<img width="1042" alt="Screen Shot 2021-10-08 at 10 01 58 AM" src="https://user-images.githubusercontent.com/5572470/136589014-6cf27b87-8538-4b9f-8266-6834d4746358.png"> 

Any suggestions?

**Comment by duncanmcclean** [comment]
Any console errors?

**Reply by brandon-9** [reply]
Only thing I see is a warning in console is: Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.

Nothing in the laravel log

**Comment by duncanmcclean** [comment, marked as answer]
For reference: bug report opened for this Discussion #4431
